resources/zib/zib-DispenseRequest.xml

```diff
@@ -0,0 +1,246 @@
+<?xml version="1.0" encoding="utf-8"?>
+<StructureDefinition xmlns="http://hl7.org/fhir">
+  <id value="zib-DispenseRequest" />
+  <url value="http://nictiz.nl/fhir/StructureDefinition/zib-DispenseRequest" />
+  <name value="ZibDispenseRequest" />
+  <title value="zib DispenseRequest" />
+  <status value="draft" />
+  <publisher value="Nictiz" />
+  <contact>
+    <name value="Nictiz" />
+    <telecom>
+      <system value="email" />
+      <value value="info@nictiz.nl" />
+      <use value="work" />
+    </telecom>
+  </contact>
+  <description value="The dispense request is the request of a prescriber to the pharmacist to dispense medication(s) to the patient to support current medication agreements. The prescriber asks them to dispense a certain amount of medicine or to dispense medicine(s) for a period of use." />
+  <purpose value="This MedicationRequest resource represents the Dutch [zib ('Zorginformatiebouwsteen', i.e. Health and Care Information Model) DispenseRequest v1.0.3(2020EN)](https://zibs.nl/wiki/DispenseRequest-v1.0.3(2020EN))." />
+  <copyright value="Copyright and related rights waived via CC0, https://creativecommons.org/publicdomain/zero/1.0/. This does not apply to information from third parties, for example a medical terminology system. The implementer alone is responsible for identifying and obtaining any necessary licenses or authorizations to utilize third party IP in connection with the specification or otherwise." />
+  <fhirVersion value="4.0.1" />
+  <mapping>
+    <identity value="zib-DispenseRequest-v1.0.3-2020EN" />
+    <uri value="https://zibs.nl/wiki/DispenseRequest-v1.0.3(2020EN)" />
+    <name value="zib DispenseRequest-v1.0.3(2020EN)" />
+  </mapping>
+  <mapping>
+    <identity value="zib-timeinterval-v1.0-2020EN" />
+    <uri value="https://zibs.nl/wiki/TimeInterval-v1.0(2020EN)" />
+    <name value="zib TimeInterval-v1.0(2020EN)" />
+  </mapping>
+  <kind value="resource" />
+  <abstract value="true" />
+  <type value="MedicationRequest" />
+  <baseDefinition value="http://hl7.org/fhir/StructureDefinition/MedicationRequest" />
+  <derivation value="constraint" />
+  <differential>
+    <element id="MedicationRequest">
+      <path value="MedicationRequest" />
+      <short value="DispenseRequest" />
+      <definition value="The dispense request is the request of a prescriber to the pharmacist to dispense medication(s) to the patient to support current medication agreements. The prescriber asks them to dispense a certain amount of medicine or to dispense medicine(s) for a period of use." />

```

Comment:
Why is the zib definition placed here?

Reply:
Same reason as https://bits.nictiz.nl/browse/ZIB-1452. Will be fixed in the zib and I will fix it in the profile.